webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -414,7 +417,7 @@ export default class SuggestionBox extends React.PureComponent {
         // set the caret position after the next rendering
         window.requestAnimationFrame(() => {
             if (textbox.value === newValue) {
-                Utils.setCaretPosition(textbox, prefix.length + term.length + 1);
+                Utils.setCaretPosition(textbox, prefix.length + term.length + suffix.length + 1);
```

Comment:
Similarly, this would make it so that, in cases where we keep the suffix, the cursor would move to the end of the textbox instead of after the autocompleted text